README.md

```diff
@@ -1,3 +1,5 @@
+<img  src="logo.png" height=256 weight=256 alt="Logo"/>
```

Comment:
use markdown style to add images.

Reply:
Removed html syntax and added pure markdown for image